index.js

```diff
@@ -378,12 +378,15 @@ class SourcebitDataClient {
         return fse.readJson(DEFAULT_FILE_CACHE_PATH);
     }
 
+    async getStaticPages() {
+      const data = await this.getData();
+      let paths = _.map(data.pages, (page) => page);
+      return paths;
+  }
+
     async getStaticPaths() {
         const data = await this.getData();
         let paths = _.map(data.pages, (page) => page.path);
-        if (process.env.NODE_ENV === 'development') {
-            paths = paths.concat(_.map(paths, (pagePath) => pagePath + (pagePath !== '/' ? '/' : '')));
-        }
```

Comment:
@JugglerX any reason you removed this.
Originally I've added this because we had a bug:

bug: https://www.notion.so/stackbit/Studio-When-navigating-to-a-page-it-loads-twice-76d463bb0ded413c8da7767172af8470#08fc4d6ad4804b3e8a8c32f1e8a33baa

commit: https://github.com/stackbit/sourcebit-target-next/commit/e77a66d6dfcdc887a510697dcdd3a1b6707e60b0